clearing_operations/scripts/GRGUtilities.py

```diff
@@ -317,40 +311,368 @@ def RotateXY(x, y, xc=0, yc=0, angle=0, units="DEGREES"):
 
         return pivot_point
 
-def ColIdxToXlName(index):
-    ''' Converts an index into a letter, labeled like excel columns, A to Z, AA to ZZ, etc. '''
-    if index < 1:
-        raise ValueError("Index is too small")
-    result = ""
-    while True:
-        if index > 26:
-            index, r = divmod(index - 1, 26)
-            result = chr(r + ord('A')) + result
+
+def GRGFromArea(AOI,
+                cellWidth,
+                cellHeight,
+                cellUnits,
+                labelStartPos,
+                labelStyle,
+                outputFeatureClass):
+    '''Create Gridded Reference Graphic (GRG) from area input.'''
+
+    # AOI = arcpy.GetParameterAsText(0)
+    # cellWidth = arcpy.GetParameterAsText(1)
+    # cellHeight = arcpy.GetParameterAsText(2)
+    # cellUnits = arcpy.GetParameterAsText(3)
+    # labelStartPos = arcpy.GetParameterAsText(4)
+    # labelStyle = arcpy.GetParameterAsText(5)
+    # outputFeatureClass = arcpy.GetParameterAsText(6)
+
+    fishnet = os.path.join("in_memory", "fishnet")
+    DEBUG = True
+    # GLOBALS
+    mxd = None
+    df = None
+    aprx = None
+    mapList = None
+
+    try:
+        #UPDATE
+        gisVersion = arcpy.GetInstallInfo()["Version"]
+        appEnvironment = Utilities.GetApplication()
+        if DEBUG == True: arcpy.AddMessage("App environment: " + appEnvironment)
+
+        isPro = False
```

Comment:
isPro - no longer used

Reply:
**isPro** showed up in a few places. removed all.